model_compression_toolkit/core/common/quantization/quantization_params_generation/error_functions.py

```diff
@@ -371,13 +374,62 @@ def _get_sliced_histogram(bins: np.ndarray,
     return bins_subset, counts_subset
 
 
+def _compute_hessian_for_hmse(node, hessian_info_service: HessianInfoService,
```

Comment:
The type hint is missing due to import issues?
Also, start each argument in a new line, please.

Reply:
Yes, there were several places where I couldn't import BaseNode without getting into circular dependency issues.